### PR TITLE
Structure for the new edX Insights guide

### PR DESCRIPTION
**Pull request by lamagnifica**
@mhoeber please take a look for any glaring or hard to correct missteps
@clintonb FYI

### CODE REVIEW

File: analytics_dashboard/analytics_dashboard/settings/local.py [hunk 1]
```diff
@@ -86,4 +86,4 @@
 ENABLE_AUTO_AUTH = True
 
 # Uncomment the line below to avoid having to worry about course permissions
-# ENABLE_COURSE_PERMISSIONS = False
```
Comment (clintonb) [inline analytics_dashboard/analytics_dashboard/settings/local.py:89]: This should be reversed.

**Comment by clintonb** [comment]
Would you add a file, `requirements/docs.txt` that has Spinx and whatever else is needed to build the documentation?

File: docs/en_us/shared/preface.rst [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+
+.. _Building and Running an edX Course: http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/
+.. _edX Course Staff Release Notes: http://edx.readthedocs.org/projects/edx-release-notes/en/latest/
+.. _Creating a Peer Assessment: http://edx.readthedocs.org/projects/edx-open-response-assessments/en/latest/
+.. _edX Research Guide: http://edx.readthedocs.org/projects/devdata/en/latest/
+.. _newsletters: http://edx.us5.list-manage.com/subscribe?u=1822a33c054dc20e223ca40e2&id=aba723f1aa 
+.. _release: http://edx.us5.list-manage2.com/subscribe?u=1822a33c054dc20e223ca40e2&id=83e46bd293
+.. _openedx-studio: http://groups.google.com/forum/#!forum/openedx-studio
+.. _Twitter:  http://twitter.com/edXstatus/status/475026709256101888
```
Comment (clintonb) [inline docs/en_us/shared/preface.rst]: Link to http://twitter.com/edXstatus instead of a specific status (in this case one from June).

**Comment by clintonb** [comment]
Nothing here actually relates to Insights. Is this commit simply to get the docs directory setup?

**Comment by lamagnifica** [comment]
@clintonb , yes. DOC-856.

**Comment by clintonb** [comment]
@lamagnifica Got it. If you can revert the change to local settings and add requirements, this is good to go.

File: analytics_dashboard/analytics_dashboard/settings/local.py (hunk 1 above)
Comment (lamagnifica) [inline analytics_dashboard/analytics_dashboard/settings/local.py:89]: done

File: docs/en_us/shared/preface.rst (hunk 2 above)
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst]: done

**Comment by lamagnifica** [comment]
@clintonb , there is an existing edx-analytics-dashboard/requrements/doc.txt file that contains this information:
  # Dependencies for building documentation
  -r base.txt
  Sphinx==1.2.1
Is that what you're looking for, or should I add more to it?

**Comment by clintonb** [comment]
@lamagnifica That is exactly what I am looking for (but clearly didn't actually look for). If you can add `sphinx_rtd_theme==0.1.6` it should be all set.

**Comment by mhoeber** [comment]
:+1:

**Comment by clintonb** [comment]
:+1:

**Pull request merged**